EthOn_glossary.md

```diff
@@ -62,27 +62,18 @@ A scalar value equal to the number of ancestor Blocks. The genesis Block has a n
 [`ethon:blockSize`](http://ethon.consensys.net/blockSize)   
 The size of the the Block header in RLP format in bytes.
 ## C
-### Call Contract Message
-[`ethon:CallContractMsg`](http://ethon.consensys.net/CallContractMsg)   
-A Call Contract Message is a Contract Message that calls a function in another Contract.
-### Call Transaction
-[`ethon:CallTx`](http://ethon.consensys.net/CallTx)   
-A type of Transaction that is directed towards a Contract Account and calls a method in the Contract's code.
+### Call Message
+[`ethon:CallMsg`](http://ethon.consensys.net/CallMsg)   
+A type of Message that is directed towards a Contract Account and calls a method in the Contract's code.
```

Comment:
Suggestion
A type of Message that is directed towards a Contract Account ~~and~~ that calls a method in the Contract's code.